drivers/SmartThings/matter-pump/profiles/pump-level.yml

```diff
@@ -6,6 +6,10 @@ components:
     version: 1
   - id: switchLevel
     version: 1
+    config:
+      values:
+        - key: "level.value"
+          range: [1, 100]

```

Comment:
From Presentation team, I heard and checked when embedded config is applied, metadata cannot be applied.